src/helpers/ipfs-daemon.js

```diff
@@ -45,7 +45,6 @@ const isIPFSCORS = async (ipfsRpc) => {
     ${IPFSCORS.map(({ key, value }) => {
       return `${key}: ${value}`
     }).join('\n    ')}`)
-    process.exit()
```

Comment:
Just wanted to highlight this in case we needed it.

Reply:
I've added it in the handler where this task is called, as we cannot do both throw and exit